Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -116,100 +107,16 @@ class TabsButton: UIButton {
-
-            self.clonedTabsButton = newTabsButton
-            newTabsButton.frame = self.bounds
-            newTabsButton.addTarget(self, action: #selector(cloneDidClickTabs), for: .touchUpInside)
-            newTabsButton.countLabel.text = countToBe
-            newTabsButton.accessibilityValue = countToBe
-            newTabsButton.insideButton.frame = self.insideButton.frame
-            newTabsButton.snp.removeConstraints()
-            self.addSubview(newTabsButton)
-            newTabsButton.snp.makeConstraints { make  in
-                make.center.equalTo(self)
-            }
-
-            // Instead of changing the anchorPoint of the CALayer, lets alter the rotation matrix math to be
-            // a rotation around a non-origin point
-            let frame = self.insideButton.frame
-            let halfTitleHeight = frame.height / 2
-            var newFlipTransform = CATransform3DIdentity
-            newFlipTransform = CATransform3DTranslate(newFlipTransform, 0, halfTitleHeight, 0)
-            newFlipTransform.m34 = -1.0 / 200.0 // add some perspective
-            newFlipTransform = CATransform3DRotate(newFlipTransform, CGFloat(-(Double.pi / 2)), 1.0, 0.0, 0.0)
-            newTabsButton.insideButton.layer.transform = newFlipTransform
-
-            var oldFlipTransform = CATransform3DIdentity
-            oldFlipTransform = CATransform3DTranslate(oldFlipTransform, 0, halfTitleHeight, 0)
-            oldFlipTransform.m34 = -1.0 / 200.0 // add some perspective
-            oldFlipTransform = CATransform3DRotate(oldFlipTransform, CGFloat(-(Double.pi / 2)), 1.0, 0.0, 0.0)
-            
-            let animate = {
-                newTabsButton.insideButton.layer.transform = CATransform3DIdentity
-                self.insideButton.layer.transform = oldFlipTransform
-                self.insideButton.layer.opacity = 0
-            }
-            
-            let completion: (Bool) -> Void = { completed in
-                let noActiveAnimations = self.insideButton.layer.animationKeys()?.isEmpty ?? true
-                if completed || noActiveAnimations {
-                    newTabsButton.removeFromSuperview()
-                    self.insideButton.layer.opacity = 1
-                    self.insideButton.layer.transform = CATransform3DIdentity
-                }
-                self.accessibilityLabel = Strings.Show_Tabs
-                self.countLabel.text = countToBe
-                self.accessibilityValue = countToBe
-            }
-            if animated {
-                UIView.animate(withDuration: 1.5, delay: 0, usingSpringWithDamping: 0.5, initialSpringVelocity: 0.0, options: [], animations: animate, completion: completion)
-            } else {
-                completion(true)
-            }
-        }
-    }
-    @objc func cloneDidClickTabs() {
-        sendActions(for: .touchUpInside)
+        currentCount = count
+        self.accessibilityLabel = Strings.Show_Tabs
```

Comment:
Now that the button is not being replaced, we should be able to move this, this is just setting to constant.

Reply:
Moved